backend/src/data/repositories/master/master.repository.ts

```diff
@@ -17,18 +21,32 @@ class Master {
     return masters.map(Master.modelToEntity);
   }
 
-  create(master: MasterEntity): Promise<void> {
-    return this.#MasterModel
+  async getByFilter(
+    filterName: Filter,
```

Comment:
since we need need get use by name, we should have two separate methods
`getMasterById` and `getMasterByEmail`